packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1357,6 +1358,18 @@ export function catalogServiceBuilder(
       );
       assertTenantKindExists(tenant);
 
+      const riskAnalysisWithSameName = eservice.data.riskAnalysis.find(
+        (ra: RiskAnalysis) =>
+          ra.name.toLowerCase === eserviceRiskAnalysisSeed.name.toLowerCase
+      );
+
+      if (riskAnalysisWithSameName !== undefined) {
```

Comment:
```suggestion
      const isDuplicateRiskAnalysis = eservice.data.riskAnalysis.some(
        (ra: RiskAnalysis) =>
          ra.name.toLowerCase() === eserviceRiskAnalysisSeed.name.toLowerCase()
      );

      if (isDuplicateRiskAnalysis) {
```
You can simplify this by using `.some` which returns a boolean; `toLowerCase` needs its brackets in order to be called